packages/frontend/src/scripts/configureTooltips.ts

```diff
@@ -102,6 +102,65 @@ export function configureTooltips() {
   }
 }
 
+export function testConfigureTooltipsAndShow() {
```

Comment:
This is copy-pasta from above but extracting the `show` function would mean that there is some state executed on import. What do you think is better, copy-pasta or state-execution on import? If you have some solution that does not have any of these problems I'm open for suggestions :smiley: